<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogProduce</class>
 <widget class="QDialog" name="DialogProduce">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/pengui/icons/pengui.svg</normaloff>:/pengui/icons/pengui.svg</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>540</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCommand">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>761</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>generated command to produce ISO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRun">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>run or copy ib ckuobiard the generated command</string>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGenerate">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>generate the command</string>
   </property>
   <property name="text">
    <string>&amp;Generate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHelp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>run or copy ib ckuobiard the generated command</string>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTheme">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>206</y>
     <width>73</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select the desired theme. To download themes: eggs wardrobe get</string>
   </property>
   <property name="text">
    <string>Theme</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFilters">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>128</y>
     <width>63</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Filters are used to remove private or unnecessaries informations from the image, note exclude.list.template is always applied</string>
   </property>
   <property name="text">
    <string>Filters</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCompression">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>167</y>
     <width>137</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>You can choose from three types of compression: fast (default), standart and max</string>
   </property>
   <property name="text">
    <string>Compression</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCompression">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>167</y>
     <width>621</width>
     <height>33</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select compression</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>151</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QComboBox" name="comboBoxTheme">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>581</width>
     <height>33</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select theme</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>271</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBoxCustom">
      <property name="toolTip">
       <string>Add exclude.list.custom to the /etc/penguins-eggs.d/exclude.list</string>
      </property>
      <property name="text">
       <string>custom</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="checkBoxHomes">
      <property name="toolTip">
       <string>Add exclude.list.homes to the /etc/penguins-eggs.d/exclude.list</string>
      </property>
      <property name="text">
       <string>homes</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="checkBoxUsr">
      <property name="toolTip">
       <string>Add exclude.list.usr to the /etc/penguins-eggs.d/exclude.list</string>
      </property>
      <property name="text">
       <string>usr</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>563</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="checkBoxScript">
      <property name="toolTip">
       <string>It only creates the scripts necessary to create the ISO, but does not create it.</string>
      </property>
      <property name="text">
       <string>Script</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxClone">
      <property name="toolTip">
       <string>Performs a complete clone of the system by also copying users and their data</string>
      </property>
      <property name="text">
       <string>Clone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxUnsecure">
      <property name="toolTip">
       <string>Selecting unsecure will also copy the data in /root to the ISO</string>
      </property>
      <property name="text">
       <string>Unsecure</string>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxCryptedClone">
      <property name="toolTip">
       <string>Performs a complete clone of the system by also copying the users and their data that will be encrypted in a LUKS volume inside the ISO </string>
      </property>
      <property name="text">
       <string>Crypted Clone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxYolk">
      <property name="toolTip">
       <string>Generates or regenerates a local repository /var/local/yolk to allow installation even in the absence of the Internet.</string>
      </property>
      <property name="text">
       <string>Yolk</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelAddons">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>12</y>
     <width>78</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>configures the addon and creates a link to the selected one on the desktop</string>
   </property>
   <property name="text">
    <string>Addons</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAddons">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>641</width>
     <height>33</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select addons to insert</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrefix">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>641</width>
     <height>33</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>prefix</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxPrefix">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>52</y>
     <width>84</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Prefix eg: egg-of_DISTRONAME-CODENAME-</string>
   </property>
   <property name="text">
    <string>Prefix</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxBasename">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>91</y>
     <width>136</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Basename (usually hostname)</string>
   </property>
   <property name="text">
    <string>Basename</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBasename">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>90</y>
     <width>621</width>
     <height>33</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>basename</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogProduce</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogProduce</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
